crates/junction-api/src/http.rs

```diff
@@ -763,7 +763,8 @@ mod tests {
         let test_json = json!({
             "codes":[ 1, 2 ],
             "attempts": 3,
-            "backoff": "1m"
+            // NOTE: serde will happily read an int here, but Duration serializes as a float
+            "backoff": 60.0,
```

Comment:
Should it?

Reply:
it's the best of a bunch of mediocre options right now. serializing as an int number of seconds, we risk loosing precision.

Reply:
We chatted about this in person. Float is being used to represent sub second durations.